features/localization/managing_countries/adding_country.feature

```diff
@@ -0,0 +1,16 @@
+@addressing
+Feature: Adding country
+    In order to sell my goods to different countries
+    As an Administrator
+    I want to add new country to the store
+
+    Background:
+        Given I am logged in as administrator
+
+    @ui
+    Scenario: Adding country
+        Given I want to create new country
+        When I choose "France"
+        And I add it
+        Then I should be notified about success
+        And this country should appear in the store
```

Comment:
``in the store`` is ok, we discussed that "registry" makes sense for shipments, orders, but here we add available country to the store as a whole.